patches/v2024.5.0.patch

```diff
@@ -557,8 +557,7 @@ index c2eb37eea5..2b5ac93392 100644
 +    //
 +    // We want to get to just `https://vaultwarden.example.com/base/path`.
 +    let baseUrl = this.win.location.href;
-+    baseUrl = baseUrl.replace(/#.*/, ""); // Strip off `#` and everything after.
```

Comment:
I think you could simply replace this with
```
baseUrl = baseUrl.replace(/[#?].*/, ""); // Strip off `#` or `?` and everything after.
```
The comments should also mention the form `https://vaultwarden.example.com/base/path/?queryParam=...` if that's now a possibility.